rhscl-postgresql-apb/roles/rhscl-postgresql-apb-openshift/README

```diff
@@ -0,0 +1,41 @@
+Postgresql-Demo-APB-Openshift
```

Comment:
I think that README was created by ansible-container. I can nuke it. I don't think we need it.